list_priorities.py

```diff
@@ -1,47 +1,48 @@
 #!/usr/bin/env python3
 
 import os
-import yaml
+import sys
 
+import yaml
+import yamlinclude
 
-def main():
-    priorities = get_priorities()
-    for key in sorted(priorities.keys()):
-        print(key)
-        for value in sorted(priorities[key]):
-            print(' ', value)
 
+DIRECTORIES = ['foxy', 'galactic', 'humble', 'rolling']
```

Comment:
If we're going to do this we should really be parsing the index to extract the rosdistro names and use them as directories, or even fetch the target YAML files directly.

Reply:
100% agreed; I didn't even think of that.  See https://github.com/ros2/ros_buildfarm_config/pull/234 , which implements this.